api/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -680,6 +680,14 @@ type AdmissionControllerFeatureConfig struct {
 	// +optional
 	Enabled *bool `json:"enabled,omitempty"`
 
+	// AdmissionControllerValidationConfig contains Admission Controller validation configurations.
+	// +optional
+	AdmissionControllerValidationConfig *AdmissionControllerValidationConfig `json:"validation,omitempty"`
+
+	// AdmissionControllerMutationConfig contains Admission Controller mutation configurations.
+	// +optional
+	AdmissionControllerMutationConfig *AdmissionControllerMutationConfig `json:"mutation,omitempty"`
```

Comment:
could we shorten the parameter names?

```suggestion
	Mutation *AdmissionControllerMutationConfig `json:"mutation,omitempty"`
```
OR perhaps,
```suggestion
	MutatingWebhook *AdmissionControllerMutatingWebhookConfig `json:"mutatingWebhook,omitempty"`
```

I'm not sure which would fit the use case better

Reply:
I think that the first one is best: https://github.com/DataDog/datadog-operator/pull/1461/commits/7b583b5c4340a04fdece995c10a3abe87281aab9 👍